Add case management images to docs (#21636)
DataDog/documentation: content/en/service_management/case_management/create_case.md

| Event Management (raw events) | Click into an event to open its side panel. Click the **Escalate** dropdown menu and select **Create a case**. |
| Cloud Cost Management | Click into a cost recommendation to open its side panel. Then, click **Create case**. |
| Sensitive Data Scanner | Click **Create case** next to a Sensitive Data Scanner issue.  |
| Slack  | Click the **Create Case** button under a monitor notification in Slack.  |

## Automatic case creation

Configure the following products to automatically create cases:
| Product | Instructions    | 
| ------  | ----------- | 

> | Monitors | When creating a monitor, include `@case-{project_handle}` in the **Notify your team** or  **Say what's happening** section. Cases are automatically created when the monitor transitions to a different status. To only create cases for certain monitor transitions, use [conditional variables][3]. As an example, to create cases only when a monitor triggers, wrap the `@case` mention with `{{#is_alert}}` and `{{/is_alert}}`.  <br><br> Navigate to [Project Settings][4] > Integrations > Datadog Monitors and then toggle it on to get your @case-<project_handle>. |

Reviewer: ```suggestion
| Monitors | When creating a monitor, include `@case-{project_handle}` in the **Notify your team** or  **Say what's happening** section. Cases are automatically created when the monitor transitions to a different status. To only create cases for certain monitor transitions, use [conditional variables][3]. As an example, to create cases only when a monitor triggers, wrap the `@case` mention with `{{#is_alert}}` and `{{/is_alert}}`.  <br><br> Navigate to the [Project Settings page][4], click **Integrations** > **Datadog Monitors**, and click on the toggle to get your @case-<project_handle>. |
```
